exodus/analysis_query/templates/query_error.html

```diff
@@ -1,4 +1,4 @@
-{% extends "base.html"%}
+{% extends "base/base.html"%}
```

Comment:
Why did every file need to be in the directory `templates/base`?
Couldn't they be directly in `templates`?

I'm fine with this if there is a reason, just I'm not seeing it now :)

Reply:
To override templates, we have to put them in a Django app. It's what I did + follow the Django namespace recommendations.